.golangci.yml

```diff
@@ -1,4 +1,9 @@
 linters-settings:
+  gosec:
+    # Currently excluding rule G602 due to issue slowing down execution
+    # https://github.com/securego/gosec/issues/1010
+    excludes:
```

Comment:
It might be worth noting in a comment that this rule currently has some soundness bugs causing false positives: see https://github.com/securego/gosec/issues/1005.

Reply:
I'll add it in as a thing to verify if we ever want to re-enable this rule after https://github.com/securego/gosec/issues/1010 is fixed!

Reply:
354e52006dd9dde59fb6a640ca298336aef3d885